builder/dockerfile/evaluator_unix.go

```diff
@@ -4,6 +4,6 @@ package dockerfile
 
 // platformSupports is a short-term function to give users a quality error
 // message if a Dockerfile uses a command not supported on the platform.
-func platformSupports(command string) error {
+func platformSupports(command interface{}) error {
```

Comment:
Although it seems more logical to have platform support checks at this package level (the instructions package is only there for parsing and typed representation. it is the dispatcher itself that should know which command is supported or not on the target platform)

I don't think we can implement interfaces for a type living in a different package, can we ?